src/main/java/com/google/sps/data/RandomPlacesSorter.java

```diff
@@ -0,0 +1,30 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.List;
+import java.util.Collections;
+
+/** 
+ * A PlacesSorter that sorts places randomly. 
+ */
+class RandomPlacesSorter implements PlacesSorter {
+
+    @Override
+    public List<Place> sortPlaces(List<Place> places) {
+      Collections.shuffle(places);
```

Comment:
This changes your actual input (which is rarely done). My first advice would be making a copy and shuffle that one, leaving the origin untouched.
However, in Google we have Immutable types for that purpose exactly (to keep us from changing input sources), and I now see it's open source as well: https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableList.html. Please change your APIs to use the Immutable types instead. This is a better practice overall.